tests/php/test-class-dev-tools-user-access.php

```diff
@@ -39,6 +39,52 @@ public function setUp() {
 	public function test_register() {
 		$this->dev_tools_user_access->register();
 		$this->assertEquals( 10, has_action( 'rest_api_init', [ $this->dev_tools_user_access, 'register_rest_field' ] ) );
+		$this->assertEquals( 10, has_action( 'personal_options', [ $this->dev_tools_user_access, 'print_personal_options' ] ) );
+		$this->assertEquals( 10, has_action( 'personal_options_update', [ $this->dev_tools_user_access, 'update_user_setting' ] ) );
+		$this->assertEquals( 10, has_action( 'edit_user_profile_update', [ $this->dev_tools_user_access, 'update_user_setting' ] ) );
+	}
+
+	/**
+	 * Tests DevToolsUserAccess::is_user_enabled
+	 *
+	 * @covers DevToolsUserAccess::is_user_enabled
+	 */
+	public function test_is_user_enabled() {
+		$admin_user  = $this->factory()->user->create_and_get( [ 'role' => 'administrator' ] );
+		$editor_user = $this->factory()->user->create_and_get( [ 'role' => 'editor' ] );
```

Comment:
The `factory` method is actually static.

```suggestion
		$admin_user  = self::factory()->user->create_and_get( [ 'role' => 'administrator' ] );
		$editor_user = self::factory()->user->create_and_get( [ 'role' => 'editor' ] );
```

Reply:
I wonder why it isn't complaining.

Reply:
Fixed in 947a56fcd522c1e55bd8d48a841b9b2a013e8fed across the codebase.